west.yml

```diff
@@ -154,7 +154,7 @@ manifest:
     - name: matter
       repo-path: sdk-connectedhomeip
       path: modules/lib/matter
-      revision: e3dd453b9bd82ee33b258cd2be94abeca9ba62c0
+      revision: pull/395/head
```

Comment:
324ff7699e65c2120af6a6c665b6ccf63cda71d9, https://github.com/nrfconnect/sdk-connectedhomeip/commit/324ff7699e65c2120af6a6c665b6ccf63cda71d9

Reply:
@jfischer-no  can you double check if all is good (i hope) and merge?